### DISCUSSION TITLE
Listbox state management problem - please help

### DISCUSSION THREAD

**Discussion by pivotpointux** [tailwindlabs/headlessui#3530] [Help]
We are using the Headless UI Listbox component and we can't find a way to easily control the selected item state externally via prop.

How can we do this? If not, then what do you suggest?

Thanks in advance!

**Comment by pivotpointux** [comment]
@adamwathan wondering if you might know about this please? Thanks in advance!

**Comment by hmaesta** [comment]
Related to https://github.com/tailwindlabs/headlessui/discussions/975
